RELEASE_PROCESS.md

```diff
@@ -20,92 +40,278 @@ Regular releases concern only major and minor releases. We do not decide up fron
-- First, changes are merged into the `spec` repository, then `spec-json-schemas` and then in `parser-js`.
+As the release gets closer to being ready, it is helpful to remind the community that the release is on the way.
 
-### Next steps
+This can be done:
+- in [Slack](https://www.asyncapi.com/slack-invite)
+- at [community meetings](https://github.com/asyncapi/community/issues?q=is%3Aissue+is%3Aopen+label%3Ameeting)
 
-- Update previously created draft of release notes with information about the new feature. Collaborate closely with feature contributors,
-- Notify maintainers of the following repositories that the first feature is merged and that release will be produced and therefore they need to start preparing for it:
+Including a link to the [release issue](#step-2---create-a-release-issue) is a good way to let the community see the progress that has been made so far.
+
+
+### Step 12 - reviews
+
+At least one [code owner](#code-owners) must approve the [release pull requests](#step-7---create-draft-pull-requests) in all related [repositories](#repositories).
+
+
+### Step 13 - release candidates
+
+_how is this done?_
+
+Example release candidate
+https://github.com/asyncapi/spec-json-schemas/pull/151
+
+
+### Step 14 - merge the release branches
+
+Once everything is ready, it is time to merge the [release branches](#step-3---create-release-branches) using the [draft pull requests prepared earlier](#step-7---create-draft-pull-requests).
+
+Merging can only be done by [code owners](#code-owners).
+
+Pull requests must have a `feat: ` prefix that, after the merge, produces a release candidate with a minor version update. The major version must have `feat!: ` prefix.
+
+Pull requests in the `parser-js` can be merged only if it uses the release candidate of `@asyncapi/specs` package produced after merge of a pull request in `spec-json-schemas`.
+
+First, changes are merged into the `spec` repository, then `spec-json-schemas` and then in `parser-js`.
+
+Release means merge of pull requests created from a release branch against the master branch. First, changes are merged into the `spec` repository, then `spec-json-schemas` and at the end in `parser-js`. Like in the case of the merge of release candidates, a pull request in `parser-js` can be merged only if it uses the final release of the new `@asyncapi/specs` package.
+
+
+### Step 15 - publish releases
+
+The [release coordinator](#%22release-coordinator%22) should ask the [code owners](#code-owners) for each repository to create the release in Github, using the [release notes they have prepared](#step-10---prepare-release-notes).
+
+
+### Step 16 - notify tool maintainers
+
+The [release coordinator](#%22release-coordinator%22) should notify maintainers of the following repositories that the first feature is merged and that release will be produced and therefore they need to start preparing for it:
```

Comment:
we could do this also when release cycle is started, to let folks know upfront, and what issue to follow to see progress

Reply:
you can do it by basically mentioning codeowners in the release progress issue